investpy/__init__.py

```diff
@@ -771,7 +832,8 @@ def get_equity_company_profile(equity, country='spain', language='english'):
         req = requests.get(url, headers=head)
```

Comment:
All the changes done not related with `cross_currencies` should not be included since the previous code was properly formatted and the published changes are just modifications on how the data is tabulated or presented to the programmer. So **just the additions related to cross currencies data retrieval will be taken into consideration**, unless changes in any other function improves it.